types/next-auth/providers.d.ts

```diff
@@ -59,9 +64,10 @@ export interface DefaultProviders {
     Twitter: Twitter;
     VK: VK;
     Yandex: Yandex;
+    Zoho: Zoho;
 }
 
-export type Providers = Array<ReturnType<DefaultProviders[keyof DefaultProviders]>>;
+export type Providers = Array<ReturnType<DefaultProviders[keyof DefaultProviders]> | Provider<string, string>>;
```

Comment:
There's already [another PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52007/files#diff-3466241fb72d7fa4d7807039e1ab0148e9d4326da0dd38d7ad6a3d1014fc50a0) ready ✅ that introduces the notion of a **custom Provider**. Can we undo this bit (_as it's already addressed there_) ?

Reply:
Done. I've reverted this change.